### PR TITLE
Update util.py

### PR DESCRIPTION
**Pull request by oxfordpelican**
Fix the bug that causes InstaPy to crash when accepting the cookies

### CODE REVIEW

**Comment by timgrossmann** [comment]
@oxfordpelican Please format your code using `black -t py34` 😊

**Comment by SanielDous** [comment]
I am already using this code for my sessions but it does not fix the issue #5837 completely. I'm not sure if it is the same problem but sometime I get the following error:

```
InstaPy Version: 0.6.12
 ._.  ._.  ._.  ._.  ._.  ._.  ._.  ._. 
Workspace in use: "/home/pi/InstaPy"
OOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO
INFO [2020-11-06 16:17:20] [my.account]  Session started!
oooooooooooooooooooooooooooooooooooooooooooooooooooooo
INFO [2020-11-06 16:17:27] [my.account]  -- Connection Checklist [1/2] (Internet Connection Status)
INFO [2020-11-06 16:17:28] [my.account]  - Internet Connection Status: ok
INFO [2020-11-06 16:17:28] [my.account]  - Current IP is "123.456.789.0" and it's from "Country"
INFO [2020-11-06 16:17:28] [my.account]  -- Connection Checklist [2/2] (Hide Selenium Extension)
INFO [2020-11-06 16:17:28] [my.account]  - window.navigator.webdriver response: None
INFO [2020-11-06 16:17:28] [my.account]  - Hide Selenium Extension: ok
Issue with cookie for user my.account. Creating new cookie...
INFO [2020-11-06 16:18:16] [my.account]  Timed out with failure while explicitly waiting until visibility of element located!

**********************************************************************
If raising an issue, please also upload the file located at:
/tmp/20201106-161821.html
**********************************************************************

INFO [2020-11-06 16:18:24] [my.account]  Sessional Live Report:
        |> No any statistics to show


[Session lasted 1.28 minutes]
OOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO
INFO [2020-11-06 16:18:24] [my.account]  Session ended!
oooooooooooooooooooooooooooooooooooooooooooooooooooooooo



On error occured while running InstaPy
Error: generator didn't yield
------- FULL TRACEBACK -------
Traceback (most recent call last):
  File "insta_bot.py", line 100, in <module>
    scripts.script.run_script(tag)
  File "/home/pi/instabot/scripts/script.py", line 31, in forheroes
    with smart_run(session):
  File "/usr/lib/python3.7/contextlib.py", line 114, in __enter__
    raise RuntimeError("generator didn't yield") from None
RuntimeError: generator didn't yield

------------------------------
InstaPy Session finished!
```

**Comment by oxfordpelican** [comment]
> I am already using this code for my sessions but it does not fix the issue #5837 completely. I'm not sure if it is the same problem but sometime I get the following error:
> 
> ```
> InstaPy Version: 0.6.12
>  ._.  ._.  ._.  ._.  ._.  ._.  ._.  ._. 
> Workspace in use: "/home/pi/InstaPy"
> OOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO
> INFO [2020-11-06 16:17:20] [my.account]  Session started!
> oooooooooooooooooooooooooooooooooooooooooooooooooooooo
> INFO [2020-11-06 16:17:27] [my.account]  -- Connection Checklist [1/2] (Internet Connection Status)
> INFO [2020-11-06 16:17:28] [my.account]  - Internet Connection Status: ok
> INFO [2020-11-06 16:17:28] [my.account]  - Current IP is "123.456.789.0" and it's from "Country"
> INFO [2020-11-06 16:17:28] [my.account]  -- Connection Checklist [2/2] (Hide Selenium Extension)
> INFO [2020-11-06 16:17:28] [my.account]  - window.navigator.webdriver response: None
> INFO [2020-11-06 16:17:28] [my.account]  - Hide Selenium Extension: ok
> Issue with cookie for user my.account. Creating new cookie...
> INFO [2020-11-06 16:18:16] [my.account]  Timed out with failure while explicitly waiting until visibility of element located!
> 
> **********************************************************************
> If raising an issue, please also upload the file located at:
> /tmp/20201106-161821.html
> **********************************************************************
> 
> INFO [2020-11-06 16:18:24] [my.account]  Sessional Live Report:
>         |> No any statistics to show
> 
> 
> [Session lasted 1.28 minutes]
> OOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO
> INFO [2020-11-06 16:18:24] [my.account]  Session ended!
> oooooooooooooooooooooooooooooooooooooooooooooooooooooooo
> 
> 
> 
> On error occured while running InstaPy
> Error: generator didn't yield
> ------- FULL TRACEBACK -------
> Traceback (most recent call last):
>   File "insta_bot.py", line 100, in <module>
>     scripts.script.run_script(tag)
>   File "/home/pi/instabot/scripts/script.py", line 31, in forheroes
>     with smart_run(session):
>   File "/usr/lib/python3.7/contextlib.py", line 114, in __enter__
>     raise RuntimeError("generator didn't yield") from None
> RuntimeError: generator didn't yield
> 
> ------------------------------
> InstaPy Session finished!
> ```

Can you set headless=False and check on what page InstaPy is stuck before you get the timeout. Is this the accept cookies page?

**Comment by SanielDous** [comment]
> Can you set headless=False and check on what page InstaPy is stuck before you get the timeout. Is this the accept cookies page?

I'm running this on my raspberry pi so I don't have the possibility to run this with an UI. I could try to reproduce it on my desktop PC. But would it help to upload the mentioned file `/tmp/20201106-161821.html`?

File: instapy/util.py [hunk 1]
```diff
@@ -1917,6 +1917,14 @@ def is_page_available(browser, logger):
 @contextmanager
 def smart_run(session, threaded=False):
     try:
+        # Accept cookies quick and dirty fix.
```
Comment (elulcao) [inline instapy/util.py:1920]: Could you check if this change can be implemented in `def loging()` or `def login_user()`?
`session.login()` is used after all,  so this section can still keep shortened.
The problem is not in `smart_run` but it is faced when the user cannot log into IG.

There is also a variable like `ig_homepage` in `login_user()` that can be used to standardize the line `1921`

**Review by elulcao — CHANGES_REQUESTED** [review]

**Comment by v0lumehi** [comment]
I fixed this in my pull request:
https://github.com/timgrossmann/InstaPy/pull/5894/

**Comment by elulcao** [comment]
Changes in #5894, closing by now this PR to avoid double efforts.

**Pull request closed without merging**